src/main/resources/db/migration/v0.38.0.7__gene_expression_experiment.sql

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
No index or foreign key definitions in this migration

Reply:
Added fkeys and indexes